src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** Represents a comment in store or user page. */
+public class Comment {
+
+  private final long id;
+  private final long rating;
+  private final String drink;
+  private final String content;
+
+  public Comment(long id, long rating, String drink, String content) {
+    this.id = id;
+    this.rating = rating;
+    this.drink = drink;
+    this.content = content;
+  }
+
+  public long getId() {
+    return id;
+  }
+
+  public long getRating() {
+    return rating;
+  }
+
+  public String getDrink() {
+    return drink;
+  }
+
+  public String getContent() {
+    return content;
+  }
```

Comment:
Should we make a toString here instead of having to call all of these functions when we display a particular comment, its rating, its drink, and content?

Reply:
Right now this is being handled in `createListElement()` in review.js, thus giving us the freedom to edit how we want to present the data on HTML page. Is there a particular reason you want this to be handled in the servlet?